### PR TITLE
docs fixes

### PR DESCRIPTION
**Pull request by burtbeckwith**
asciidoc fixes
small fixes
src/main/docs code cleanup

### CODE REVIEW

File: src/main/docs/guide/cloud/cloudConfiguration.adoc [hunk 1]
```diff
@@ -113,17 +113,11 @@ The api:runtime.server.EmbeddedServerInstance[] is available through event liste
...
-[source,java]
+[source,groovy]
 ----
 @EventListener
 void onServiceStarted(ServiceReadyEvent event) {
-    ServiceInstance serviceInstance = event.getSource()
+    ServiceInstance serviceInstance = event.source
     ConvertibleValues<String> metadata = serviceInstance.metadata
 }
 ----
```
Comment (jameskleeh) [inline src/main/docs/guide/cloud/cloudConfiguration.adoc]: For inline snippets like this they should be in Java

File: src/main/docs/guide/cloud/serviceDiscovery/serviceDiscoveryRoute53.adoc [hunk 2]
```diff
@@ -52,11 +52,11 @@ aws:
...
 
 .Sample code for client
-[source,java]
+[source,groovy]
 ----
 DiscoveryClient discoveryClient = embeddedServer.applicationContext.getBean(DiscoveryClient);
-List<String> serviceIds = Flowable.fromPublisher(discoveryClient.getServiceIds()).blockingFirst();
-List<ServiceInstance> instances = Flowable.fromPublisher(discoveryClient.getInstances(serviceIds.get(0))).blockingFirst();
+List<String> serviceIds = Flowable.fromPublisher(discoveryClient.serviceIds).blockingFirst()
+List<ServiceInstance> instances = Flowable.fromPublisher(discoveryClient.getInstances(serviceIds[0])).blockingFirst()
```
Comment (jameskleeh) [inline src/main/docs/guide/cloud/serviceDiscovery/serviceDiscoveryRoute53.adoc]: Same for here, prefer Java

File: src/main/docs/guide/ioc.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 When most developers think of Inversion of Control (also known as Dependency Injection and referred to as such from this point onwards) the Spring Framework comes to mind.
 
-Micronaut takes inspiration from Spring, and in fact, the core developers of Micronaut are former SpringSource/Pivotal engineers now working for https://objectcomputing.com[OCI].
+Micronaut takes inspiration from Spring, and in fact, the core developers of Micronaut are former SpringSource/Pivotal engineers now working for https://objectcomputing.com[OCI] and https://labs.oracle.com[Oracle Labs].
```
Comment (jameskleeh) [inline src/main/docs/guide/ioc.adoc]: I think I would rather remove both companies there. Given the core developers may include other companies in the future it doesn't make sense to call out which specific ones have former springsource devs. Thoughts @jeffbrown ?

Comment (jeffscottbrown) [reply src/main/docs/guide/ioc.adoc]: Personally, I don't think mentioning that some of the current team used to work for Pivotal is terribly important.  The majority of the Micronaut core team did not work for SpringSource or Pivotal.

Comment (graemerocher) [reply src/main/docs/guide/ioc.adoc]: I think this whole paragraph should just be removed and references to Spring

**Comment by jameskleeh** [comment]
@burtbeckwith After the conflict is resolved I think its good to go

**Pull request merged**